app/create_db_tables.py

```diff
@@ -10,37 +10,38 @@
 
 def initialize_police_table():
     load_dotenv()
-    police_table = """CREATE TABLE IF NOT EXISTS police_force (
-    id SERIAL PRIMARY KEY NOT NULL,
-    dates TIMESTAMP,
-    links TEXT,
-    case_id TEXT,
-    city TEXT,
-    state TEXT,
-    title TEXT,
-    description TEXT,
-    tags TEXT,
-    force_rank TEXT,
-    confidence TEXT
-        );"""
+    # police_table = """CREATE TABLE IF NOT EXISTS police_force (
```

Comment:
When we confirm the shared architecture of our table for backend use - we should delete police_table since it does not align with the proposed architecture.

Reply:
I agree sounds good!